tests/functional/raw-node/test/routes/routeMetadata.js

```diff
@@ -43,7 +43,7 @@ describe('metadata routes with metadata mock backend', () => {
         makeMetadataRequest({
             method: 'GET',
             authCredentials: metadataAuthCredentials,
-            path: '/_/metadata/listbuckets/1',
+            path: '/_/metadata/admin/raft_sessions/1/bucket',
```

Comment:
https://github.com/scality/backbeat/blob/development/8.1/lib/clients/backbeat-2017-07-01.api.json
Would this file in backbeat have to be changed to reflect the new path names?

Reply:
Correct, I will be making the changes. It's easier this way as we don't have to write a method for each call.